plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/exception/SMExceptionTooManySessions.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.security.exception;
+
+public class SMExceptionTooManySessions extends SMException {
+
+    private final String errorType;
+
+    public SMExceptionTooManySessions(String message, String errorType) {
```

Comment:
do we really need `errorType` as parameter? 
it not a general error, and it seems we can just hardcode `tooManySessions` here